### PR TITLE
Fixed #16722, container size preventing map rotation.

### LINKED ISSUE #16722: Projection Orthographic does not rotate map, when maps container is larger.
#### Expected behaviour
Projection Orthographic should rotate world map, no matter what container height is.

#### Actual behaviour
Projection Orthographic does not rotate world map when height of container is bigger than ~680px;

#### Live demo with steps to reproduce

template: https://jsfiddle.net/RobertRochon/97y0eb2L/1/

When we have the demo `Built in projection` we had #container height set to 500px.
When height is modified to more than ~680px, rotation does not work and standard map navigation is working.

#### Product version

Highcharts Maps 9.3.2

#### Affected browser(s)
Tested on:
Google Chrome - 95.0.4638.54
Firefox - 94.0

### PR DESCRIPTION
**Pull request by DJTechnoo**
Fixed #16722, container size preventing map rotation.

### CODE REVIEW

**Comment by KacperMadej** [comment]
There is no linear relation between `minZoom` and `minPlotSize` - tested [here](https://jsfiddle.net/BlackLabel/pje8v9t0/).
There is a linear relation between `minPlotSize` and `getScale` result - see [demo](https://jsfiddle.net/BlackLabel/pje8v9t0/1/).
For changing chart size the factor is constant - [demo](https://jsfiddle.net/BlackLabel/pje8v9t0/2/).
After zooming in the scale goes up, so it would be possible to pick a threshold for a different panning mode (move or rotate).

File: ts/Maps/MapView.ts [hunk 1]
```diff
@@ -164,21 +164,24 @@ class MapView {
                 }
 
                 // Panning rotates the globe
```
Comment (TorsteinHonsi) [inline ts/Maps/MapView.ts:166]: This comment should be further down, before line 172. Otherwise the context is lost.

**Review by KacperMadej — APPROVED** [review]
A conflict to resolve before the merge. Other than that, looks good 👍

**Review by madepiet — APPROVED** [review]

File: samples/unit-tests/maps/map-navigation/demo.js [hunk 2]
```diff
@@ -190,3 +190,150 @@ QUnit.test('Map navigation button alignment', assert => {
...
+                geometry: {
+                    type: 'LineString',
+                    coordinates
+                },
+                lineWidth: y === 0 ? 2 : undefined
+            });
+        }
+
+        return data;
+    };
```
Comment (raf18seb) [inline samples/unit-tests/maps/map-navigation/demo.js:231]: Do we need a graticule in a unit test? It's additional tens of thousands of operations that maybe can be skipped?

**Review by raf18seb — CHANGES_REQUESTED** [review]

File: samples/unit-tests/maps/map-navigation/demo.js (hunk 2 above)
Comment (TorsteinHonsi) [reply samples/unit-tests/maps/map-navigation/demo.js:231]: You're right. We need something that makes the initial zoom show the full world, but it doesn't need to be this complex. Either a few lines (equator + a handful of meridians), or some wisely placed map points (poles + some around the equator?) should be enough to make the ortho projection zoom out to the full globe.

**Review by TorsteinHonsi — APPROVED** [review]
Thanks! Nice test with the partial graticule.

**Pull request merged**